diff --git a/node_modules/@chakra-ui/button/dist/cjs/button.js b/node_modules/@chakra-ui/button/dist/cjs/button.js
index de61c03..1f0d6cd 100644
--- a/node_modules/@chakra-ui/button/dist/cjs/button.js
+++ b/node_modules/@chakra-ui/button/dist/cjs/button.js
@@ -82,7 +82,7 @@ var Button = /*#__PURE__*/(0, _system.forwardRef)(function (props, ref) {
       setIsButton = _React$useState[1];
 
   var refCallback = React.useCallback(function (node) {
-    if ((node == null ? void 0 : node.tagName) !== "BUTTON") {
+    if (node && node.tagName !== "BUTTON") {
       setIsButton(false);
     }
   }, []);
diff --git a/node_modules/@chakra-ui/button/dist/esm/button.js b/node_modules/@chakra-ui/button/dist/esm/button.js
index d07f4ea..88baa13 100644
--- a/node_modules/@chakra-ui/button/dist/esm/button.js
+++ b/node_modules/@chakra-ui/button/dist/esm/button.js
@@ -62,7 +62,7 @@ export var Button = /*#__PURE__*/forwardRef((props, ref) => {
 
   var [isButton, setIsButton] = React.useState(!as);
   var refCallback = React.useCallback(node => {
-    if ((node == null ? void 0 : node.tagName) !== "BUTTON") {
+    if (node && node.tagName !== "BUTTON") {
       setIsButton(false);
     }
   }, []);
